src/fairchem/core/common/utils.py

```diff
@@ -393,7 +402,11 @@ def create_dict_from_args(args: list, sep: str = "."):
     return return_dict
 
 
-def load_config(path: str, previous_includes: list | None = None):
+def load_config(
+    path: str, previous_includes: list | None = None, include_paths: list | None = None
```

Comment:
maybe comment on the diff between paths, previous_includes and include_paths?

Reply:
also optional - add test on yml with include paths if doesnt exist, can also make task for BE :)